public/modules/core/views/invoice.client.view.html

```diff
@@ -13,32 +13,33 @@
                     No invoice found
                 </div>
                 <div class="col-sm-12" ng-if="invoices.length" style="margin-top:20px;">
+                  <div class="table-responsive">
                     <table class="table table-hover">
-                        <tbody>
-                            <tr>
-                                <th>Status</th>
-                                <th>Date</th>
-                                <th>Number</th>
-                                <th>Amount</th>
-                                <th>Payment</th>
-                                <th>Download</th>
-                            </tr>
-                            <tr ng-repeat="list in invoices" ng-if=" list.invoice.status !='pending' ">
-                                <td>
-                                    <i class="fa fa-info-circle text-warning" ng-if="list.invoice.status == 'payment_due'"></i>
-                                    <i class="fa fa-check text-success" ng-if="list.invoice.status == 'paid'"></i>
-                                    <i class="fa fa-close text-danger" ng-if="list.invoice.status == 'not_paid'"></i>
-                                </td>
-                                <td>
-                                    {{ list.invoice.end_date * 1000 | date:'MM/dd/yyyy' }}
-                                </td>
-                                <td> 
-                                    {{ list.invoice.id }}
-                                </td>
-                                <td >
-                                    {{ list.invoice.amount/100 | currency }}
-                                </td>
-                                <td class="text-muted">
+                      <tbody>
+                      <tr>
+                        <th>Status</th>
+                        <th>Date</th>
+                        <th>Number</th>
+                        <th>Amount</th>
+                        <th>Payment</th>
+                        <th>Download</th>
+                      </tr>
+                      <tr ng-repeat="list in invoices" ng-if=" list.invoice.status !='pending' ">
+                        <td>
+                          <i class="fa fa-info-circle text-warning" ng-if="list.invoice.status == 'payment_due'"></i>
+                          <i class="fa fa-check text-success" ng-if="list.invoice.status == 'paid'"></i>
+                          <i class="fa fa-close text-danger" ng-if="list.invoice.status == 'not_paid'"></i>
+                        </td>
+                        <td>
+                          {{ list.invoice.end_date * 1000 | date:'MM/dd/yyyy' }}
+                        </td>
+                        <td>
+                          {{ list.invoice.id }}
+                        </td>
+                        <td >
+                          {{ list.invoice.amount/100 | currency }}
```

Comment:
We can review code on the following iterations, this PR is intended only for styling.